backend/courses/migrations/0001_initial.py

```diff
@@ -0,0 +1,182 @@
+# Generated by Django 2.2.10 on 2020-05-25 13:56
+
```

Comment:
Don't commit migration files. Also this doesn't have  anything to do in the `fix_chat` PR